wundertools/docker/alpine-mariadb/Makefile

```diff
@@ -0,0 +1,31 @@
+IMAGE = alpine-mariadb
+
+build: conf.database conf.user conf.root.pass conf.user.pass
+	docker \
+		build \
+		--build-arg "MYSQL_ROOT_PASSWORD=$(shell cat conf.root.pass)" \
+		--build-arg "MYSQL_DATABASE=$(shell cat conf.database)" \
+		--build-arg "MYSQL_USER=$(shell cat conf.user)" \
+		--build-arg "MYSQL_PASSWORD=$(shell cat conf.user.pass)" \
+		-t "$(IMAGE)" \
+		.
+
+rmi:
+	-docker rmi $(IMAGE)
+
+clean:
+	docker ps -a --filter status="exited" --format '{{.ID}}' | xargs -r docker rm
```

Comment:
This is completely true. While building images you often get broken builds that result in exited containers. So to save manual labour just type "make clean". It assumes you have no need keep around dead containers.